index.js

```diff
@@ -1,86 +1,6 @@
-'use strict';
+import ember from './configs/ember.js';
+import javascript from './configs/javascript.js';
+import node from './configs/node.js';
+import typescript from './configs/typescript.js';
 
-const js = require('@eslint/js');
-
-module.exports = [
-  js.configs.recommended,
-  {
-    languageOptions: {
-      globals: {},
-      ecmaVersion: 2022,
-    },
-
-    rules: {
-      'comma-dangle': ['error', 'always-multiline'],
-      'eol-last': 'error',
-      eqeqeq: 'error',
-
-      indent: ['error', 2, {
-        MemberExpression: 1,
-        SwitchCase: 1,
-      }],
-
-      'max-params': ['error', 3],
-      'no-constant-condition': 'off',
-
-      'no-empty': ['error', {
-        allowEmptyCatch: true,
-      }],
-
-      'no-eval': 'error',
-      'no-restricted-globals': ['error', 'event'],
-      'object-shorthand': 'error',
-      'one-var': ['error', 'never'],
-
-      quotes: ['error', 'single', {
-        allowTemplateLiterals: true,
-        avoidEscape: true,
-      }],
-
-      'require-await': 'error',
-      semi: 'error',
-      'arrow-spacing': 'error',
-      'array-bracket-spacing': 'error',
-      'block-spacing': 'error',
-      'comma-spacing': 'error',
-      'computed-property-spacing': 'error',
-      'func-call-spacing': 'error',
-
-      'generator-star-spacing': ['error', {
-        named: 'after',
-        anonymous: 'after',
-      }],
-
-      'key-spacing': 'error',
-      'keyword-spacing': ['error', {}],
-      'no-multi-spaces': 'error',
-      'no-trailing-spaces': 'error',
-      'no-whitespace-before-property': 'error',
-      'object-curly-spacing': ['error', 'always'],
-      'rest-spread-spacing': 'error',
-      'semi-spacing': 'error',
-      'space-before-blocks': 'error',
-
-      'space-before-function-paren': ['error', {
-        anonymous: 'ignore',
-        named: 'never',
-        asyncArrow: 'always',
-      }],
-
-      'space-in-parens': 'error',
-      'space-infix-ops': 'error',
-      'space-unary-ops': 'error',
-
-      'spaced-comment': ['error', 'always', {
-        block: {
-          balanced: true,
-          exceptions: ['*'],
-        },
-      }],
-
-      'switch-colon-spacing': 'error',
-      'template-curly-spacing': ['error', 'never'],
-      'template-tag-spacing': 'error',
-      'yield-star-spacing': 'error',
-    },
-  }];
+export { ember, javascript, node, typescript };
```

Comment:
we would need a major version release, as this breaks how its imported in existing versions